### DISCUSSION TITLE
Rooted K1 Max: Timelapse Camera (select which camera should be used) not selectable.

### DISCUSSION THREAD

**Discussion by OriginalHotrodder** [Guilouz/Creality-Helper-Script-Wiki#483] [Issues]
After I rooted I was able to select and use the K1 Max timelapse function and it worked great. About a month ago after updating to 1.3.3.5 the camera selection drop down in settings was no longer selectable. I have the camera settings loaded in Moonraker.cfg correctly but there is no timelapse created since there is no camera listed in the one to use. Is there a log I can review to see what may be the issue? Thank you in advance for your assistance.

**Comment by TheWebMachine** [comment]
You are referring to Mainsail's timelapse settings section, yes?

If so, this is a known issue at this point and has been mentioned elsewhere, but not sure which component the issue is stemming from yet. 

That said, there is a workaround: install Fluidd alongside Mainsail, then you can select the camera in the dropdown in Fluidd and it sets it up correctly. You don't need to fuss with Fluidd again after that, but I left it installed on mine just in case something else in Mainsail proves funky. This worked for me on my K1C on both 1.3.3.5 and .8

**Reply by OriginalHotrodder** [reply]
Yes I am, thank you, I'll try that now.

**Reply by TheWebMachine** [reply]
Oh, and if you are using the layerheight routine for timelapse, you definitely need the per-layer gcode line in your slicer, per the timelapse documentation:

`TIMELAPSE_TAKE_FRAME`

Since it was working for you before, I suspect this is already the case or you aren't using layerheight routine. I am merely adding the above for anyone else who stumbles upon this discussion. 🤓

Another note for K1C users: **_DO NOT USE THE PARK HEAD OPTION WITH EITHER OF THE REAR CORNERS UNLESS YOU CAREFULLY ADJUST THE PRECISE PARK POSITION FURTHER AWAY FROM THE CORNERS!_** Creality "cut corners" with the space in the chamber, literally. Taking the print head to either far rear corner for a snapshot **_will_** cause the extruder body to COLLIDE with the stepper motors in the corners, which will cause a belt skip, a shift of the known extruder position in the X and Y, and ruin your print. (I have the failed prints to prove it, unfortunately.) 

This can probably be fixed in the macros for the K1C, but will need looked into.
